extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -84,6 +84,26 @@ export const profile = localize('profile', "Profile");
 export const selectConnection = localize('selectConnection', "Select connection");
 export const connection = localize('connection', "Connection");
 
+// Add Database Reference dialog strings
+
+export const addDatabaseReferenceDialogName = localize('addDatabaseReferencedialogName', "Add database reference");
+export const addDatabaseReferenceOkButtonText = localize('addDatabaseReferenceOkButtonText', "Add reference");
+export const referenceRadioButtonsGroupTitle = localize('referenceRadioButtonsGroupTitle', "Type");
+export const systemDatabaseRadioButtonTitle = localize('systemDatabaseRadioButtonTitle', "System database");
+export const dacpacText = localize('dacpacText', "Data-tier application (DAC)");
+export const dacpacPlaceholder = localize('dacpacPlaceholder', "Select DAC");
+export const loadDacpacButton = localize('loadDacpacButton', "Select dacpac");
+export const locationDropdown = localize('locationDropdown', "Location");
+export const sameDatabase = localize('sameDatabase', "Same database");
+export const differentDbSameServer = localize('differentDbSameServer', "Different database, same server");
+export const differentDbDifferentServer = localize('differentDbDifferentServer', "Different database, different server");
```

Comment:
@dzsquared any preference on what the order should be here? SSDT has it as database first in its dropdown

Reply:
@kisantia I'd prefer database stay first, as in the context of the full option list it compares to the "same database" location first, then which server as a secondary option.

Reply:
I think there's an argument there for swapping as well, with the first part of the string being the point of comparison as the increasing scope of "same db" -> "same server" -> "different server".  Bonus in that the strings are more scanable since the differentiating part then comes first.